spec/shared_examples_of_stubbed_api_requests.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+require 'support/request_handlers'
+
+shared_examples "stubbed api requests" do
+  include RequestHandlers
+
+  let(:csv_dir)       { File.dirname(__FILE__)                                            }
+  let(:float_json)    { File.read(csv_dir + '/fixtures/float_tasks_api_response.json')    }
+  let(:csv_new_body)  { File.read(csv_dir + '/fixtures/googledocs_schedule_new_body.csv') }
+  let(:incidents)     { "{\"incidents\":[]}"                                              }
```

Comment:
For uninterpolated JSON, use single quotes to prevent having to escape all of the double quotes.